spec/preservation/update_moab_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe Preservation::UpdateMoab do
+  let(:full_druid) { 'druid:bj102hs9687' }
+  let(:mock_so) { instance_double(Moab::StorageObject) }
+  let(:pres_update_moab) { described_class.new }
+
+  context '#perform' do
+    it 'calls #update_moab' do
+      mock_path = instance_double(Pathname)
+      expect(Moab::StorageServices).to receive(:find_storage_object).with(full_druid, true).and_return(mock_so)
+      expect(mock_so).to receive(:object_pathname).and_return(mock_path)
+      expect(mock_path).to receive(:mkpath)
+      expect(pres_update_moab).to receive(:update_moab).with(mock_so)
+      pres_update_moab.perform(full_druid)
+    end
+  end
+
+  context '#update_moab' do
+    let(:mock_new_version) { instance_double(Moab::StorageObjectVersion) }
+    let(:verification_result) { instance_double(Moab::VerificationResult) }
+
+    it 'verifies new version of moab' do
```

Comment:
... with Moab StorageObjectVersion.verify_version_storage ?   or `it 'calls verify_version_storage on Moab::StorageObjectVersion'` or ???    we use "verify" in so many places all over this preservation code ...